vaccine_feed_ingest/utils/match.py

```diff
@@ -220,7 +220,21 @@ def has_matching_phone_number(
         if src_phone == cand_phone:
             return True
 
-    return False
+    # Only consider phone mismatched if numbers have the same area code and do not match
+    cand_area_code = str(cand_phone.national_number)[:3]
+
+    for src_phone in src_phones:
```

Comment:
nit: should we combine this loop and the one above it?

Reply:
Yeah, these could be combined. Still need to go through all of the phone numbers though to declare a mismatch. I'll do that in another PR.